sys/net/include/rpl.h

```diff
@@ -41,7 +41,9 @@ extern "C" {
 #define CC1100_RADIO_MODE CC1100_MODE_WOR
 
 #define RPL_PKT_RECV_BUF_SIZE 16
+#ifndef RPL_PROCESS_STACKSIZE
 #define RPL_PROCESS_STACKSIZE KERNEL_CONF_STACKSIZE_MAIN
+#endif
```

Comment:
AFAIK It's just sloppy style to not have the `ifndef`s.
Also, yes, do a separate PR for these please.